src/Models/LayerModel.vala

```diff
@@ -0,0 +1,76 @@
+/*
+* Copyright (c) 2019 Alecaddd (https://alecaddd.com)
```

Comment:
`2019-2020`

Reply:
Sure

Reply:
(Also I think we should update the Copyright notice in almost any new file. Could this be done automatically? Better a separate PR)